django/core/exceptions.py

```diff
@@ -73,6 +73,11 @@ class ImproperlyConfigured(Exception):
     pass
 
 
+class InsecureFilePermissionError(Exception):
```

Comment:
Not sure a custom exception adds much value. How about something like `RuntimeError` with an informative message?